Automattic/jetpack
# VideoPress: rewrite player by using TypeScript

This is a tidying/janitorial PR. It mainly rewrites the player component using TyprScript, renames the folder name, minor jsdoc improvements, and so on.

Part of https://github.com/Automattic/jetpack/issues/27655

## Proposed changes:

* VideoPress: rewrite player by using TypeScript

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Create a video block
* Upload a video
* Confirm the player behaves as usual
* You'd like to check the Player component by using the React dev tool

<img width="1728" alt="Screen Shot 2023-03-01 at 11 04 54" src="https://user-images.githubusercontent.com/77539/222163311-946e0325-f70a-4ecb-a2c3-d20c6e23182e.png">
